cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -1459,7 +1459,7 @@ def test_json_responses(self):
         self.validate_course_xblock_info(json_response, course_outline=True)
 
     @ddt.data(
-        (ModuleStoreEnum.Type.split, 5, 5),
+        (ModuleStoreEnum.Type.split, 4, 4),
```

Comment:
Calls are decreasing because split stores all `Scope.settings` fields alongside the course structure, whereas it stores `Scope.content` in a separately versioned document. By moving `is_entrance_exam` from one to the other, you're changing whether it has to read the `definition` document for the sequence.